src/frontend/screens/Library/components/InstallModal/DownloadDialog/DLCDownloadListing.tsx

```diff
@@ -21,14 +21,18 @@ const DLCDownloadListing: React.FC<Props> = ({
     return null
   }
 
+  useEffect(() => {
+    setInstallAllDlcs(dlcsToInstall.length === DLCList.length)
+  }, [dlcsToInstall])
+
   const handleAllDlcs = () => {
     setInstallAllDlcs(!installAllDlcs)
     if (!installAllDlcs) {
       setDlcsToInstall(DLCList.map(({ app_name }) => app_name))
     }
 
     if (installAllDlcs) {
-      setDlcsToInstall([...DLCList.map(({ app_name }) => app_name)])
+      setDlcsToInstall([])
```

Comment:
do we want this change? This was intentionally changed before https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/2976

Reply:
The install all dlcs checkbox cannot be disabled. I thought it was a bug I introduced when I was splitting the components.